std/asys/FileStat.hx

```diff
@@ -0,0 +1,40 @@
+package asys;
+
+typedef FileStatData = {
+	var atime:Date;
```

Comment:
Maybe leave times in this structure as unix timestamps to avoid creation of unnecessary `Date` instances?